spring-integration-core/src/main/java/org/springframework/integration/dsl/Avro.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.integration.dsl;
+
+import org.apache.avro.specific.SpecificRecord;
+
+import org.springframework.integration.transformer.SimpleFromAvroTransformer;
+import org.springframework.integration.transformer.SimpleToAvroTransformer;
+
+/**
+ * Factory methods for Avro DSL components
+ *
+ * @author Gary Russell
+ * @since 5.2
+ *
+ */
+public final class Avro {
+
+	public static SimpleToAvroTransformer toAvro() {
+		return new SimpleToAvroTransformer();
+	}
+
+	public static SimpleFromAvroTransformer fromAvro(Class<? extends SpecificRecord> type) {
```

Comment:
Can't these two go to the `Transformers` factory?

Reply:
I did that originally but figured it would mean a non-optional dependency on Avro because of the `SpecificRecord` reference.

Reply:
OK. Can we then just not have this factory at all?
Doesn't look like it brings too much value over simple direct usage of those transformers.
Or do you have something else in your mind in regards to Avro features?